akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -187,6 +187,18 @@ object ClusterEvent {
     def getAllDataCenters: java.util.Set[String] =
       scala.collection.JavaConverters.setAsJavaSetConverter(allDataCenters).asJava
 
+    /**
+     * @return `true` if more than one more than one `Version` among the members, which
```

Comment:
"more than one" appears more than once

```suggestion
     * @return `true` if more than one `Version` among the members, which
```

Reply:
fixed